src/qml/filters/audio_lowpass/ui.qml

```diff
@@ -21,31 +21,52 @@ import QtQuick.Controls 2.12
 import QtQuick.Layouts 1.12
 import Shotcut.Controls 1.0 as Shotcut
 
-Item {
+Shotcut.KeyframableFilter {
+    property string cutoffProperty: '0'
+    property string stagesProperty: '1'
+    property string wetnessProperty: 'wetness'
     width: 350
     height: 125
+    keyframableParameters: preset.parameters
+    startValues: [0, 0, 0]
+    middleValues: [0, 0, 1]
+    endValues: [0, 0, 0]
     Component.onCompleted: {
         if (filter.isNew) {
             // Set default parameter values
-            filter.set('0', 2637)
-            filter.set('1', 1)
-            filter.set('wetness', 1.0)
+            filter.set(cutoffProperty, 2637)
+            filter.set(stagesProperty, 1)
+            filter.set(wetnessProperty, 1.0)
             filter.savePreset(preset.parameters)
         }
         setControls()
     }
 
     function setControls() {
-        sliderCutoff.value = filter.getDouble('0')
-        sliderStages.value = filter.get('1')
-        sliderWetness.value = filter.getDouble('wetness') * sliderWetness.maximumValue
+        var position = getPosition()
+        blockUpdate = true
+        sliderCutoff.value = filter.getDouble(cutoffProperty, position)
+        cutoffKeyframesButton.checked = filter.animateIn <= 0 && filter.animateOut <= 0 && filter.keyframeCount(cutoffProperty) > 0
+        sliderStages.value = filter.getDouble(stagesProperty, position)
+        stagesKeyframesButton.checked = filter.animateIn <= 0 && filter.animateOut <= 0 && filter.keyframeCount(stagesProperty) > 0
+        sliderWetness.value = filter.getDouble(wetnessProperty, position) * sliderWetness.maximumValue
+        wetnessKeyframesButton.checked = filter.animateIn <= 0 && filter.animateOut <= 0 && filter.keyframeCount(wetnessProperty) > 0
+        blockUpdate = false
+        enableControls(isSimpleKeyframesActive())
+    }
+
+    function enableControls(enabled) {
+        sliderCutoff.enabled = sliderStages.enabled = sliderWetness.enabled = enabled
+    }
 
+    function updateSimpleKeyframes() {
+        updateFilter(wetnessProperty, sliderWetness.value / sliderWetness.maximumValue, wetnessKeyframesButton, null)
```

Comment:
I decided that we need to include all keyframable params in simple params. The consistency is more important than some nifty mode of operation. Besides, with the defaults, it might still fade in and out nicely with even if you forget to update the frequency at start and end.